### ISSUE TITLE
TypeError issues with running {npm run dev} on Master Branch

### ISSUE THREAD

**Issue by ranjitarjunan** [mac-s-g/react-json-view#327]
Hi there, 

Thanks for creating this project, it's exactly what I need for my front end application at work. 
To let you  I am fairly new to front end development, so apologies if this issue is quite trivial, however have been spending two days trying to wrap my head around. 

So I cloned the master branch, and carried out the following steps as instructed in the read me: 
npm install --save-dev
npm run dev 

What renders to the screen is a blank page, and receive the following console errors: 

Uncaught TypeError: Cannot read property 'filter' of undefined
    at eval (_dispatchable.js?fbf7:31)
    at f2 (_curry2.js?af5f:28)
    at reject (reject.js?5ad5:32)
    at eval (_curry2.js?af5f:20)
    at f1 (_curry1.js?8fe3:16)
    at new FilterController (FilterController.js?4145:217)
    at react-dom.js:4698
    at measureLifeCyclePerf (react-dom.js:4479)
    at ReactCompositeComponentWrapper._constructComponentWithoutOwner (react-dom.js:4697)
    at ReactCompositeComponentWrapper._constructComponent (react-dom.js:4688)

I have probably missed a key step, but any suggestions how to rectify this issue would be greatly appreciated. 

Arj

**Comment by mac-s-g** [comment]
@ranjitarjunan can you tell me what versions of node and npm you're using?
you can check that by running `npm -v` for npm version and `node -v` for node version.
i ask because different versions can behave differently with the same `package.json`.

**Comment by ranjitarjunan** [comment]
@mac-s-g 
npm version: 6.14.10
node version: 14.15.4

**Comment by mac-s-g** [comment]
okay, i'm testing now using the following:
node: 14.15.3
npm: 6.14.9

i was able to run the dev server after a fresh install without any issues.
i'm also working in an ubuntu / linux environment.

try the following:
 - delete your node modules folder
 - do a `git pull` to make sure you have the latest code
 - install fresh without save-dev flag (just `npm install`)
 - `npm run dev`

if that doesn't resolve the issue, you're gonna need to dig a littler deeper and identify what node module the error is coming from.

based on your original error, it appears to be coming from a dependency like lodash or .  if you can identify what dependency is causing the error, you might be able to find related threads on this type of isse.  

I'll leave this issue open for now in case anyone else is having the same problem.

**Comment by ranjitarjunan** [comment]
@mac-s-g I solved the issue as such: 

(1) When specifying the custom setup for Node.js, I picked Node.js runtime instead of npm package manager. This probs solved the issue I outlined above. 
(2) I am working on Windows, so within the scripts section of Node.js, where you specified NODE_ENV, error returned as NODE_ENV is not recognised as an internal or external command. To resolve this, I executed the following command: 

npm install -g win-node-env 

I'll go ahead and close the issue. 

Cheers Arj
